pkg/buildclient/messages.go

```diff
@@ -229,8 +230,22 @@ func (m *WebsocketMessages) Recv() (<-chan *Message, func()) {
 }
 
 func (m *WebsocketMessages) Send(msg *Message) error {
-	logrus.Tracef("Send build message %s", msg)
+	logrus.Tracef("Send build message %s", redact(msg))
```

Comment:
This could probably be achieved through a custom logrus formatter as well, but I think this is concise and works well enough.

Reply:
Yeah, no need to right now unless you can think of other places that need redactions.